lib/Doctrine/DBAL/Driver/Mysqli/MysqliConnection.php

```diff
@@ -63,6 +63,19 @@ public function __construct(array $params, $username, $password, array $driverOp
 
         $this->_conn = mysqli_init();
 
+        if (isset($driverOptions['ssl_key']) &&
+            isset($driverOptions['ssl_cert']) &&
+            isset($driverOptions['ssl_ca'])
```

Comment:
@pgrau simply put them into `$params` argument instead of `$driverOptions`. `$driverOptions` is exclusively used for `\MYSQLI_*` constants so the SSL related parameters do not fit in there.

**Example**

```php
$connection = \Doctrine\DBAL\DriverManager::getConnection(
    array(
        'dbname' => 'mydb',
        'user' => 'user',
        'password' => 'secret',
        'host' => 'localhost',
        'driver' => 'mysqli',
        'ssl_key' => '/path/to/ssl_key',
        'ssl_cert' => '/path/to/ssl_key',
        'ssl_ca' => '/path/to/ssl_key',
        // ...
    )
);